crates/core/tedge_actors/src/builders.rs

```diff
@@ -102,9 +102,6 @@ pub struct NoConfig;
 /// For that purpose each source implementation defines a `Config` type parameter,
 /// and the sink has to provide the configuration value specific to its needs.
 pub trait MessageSink<M: Message, Config> {
-    /// Return the config used by this actor to connect the message source
-    fn get_config(&self) -> Config;
-
```

Comment:
All the implementations were either returning `NoConfig` or delegating some inner implementation.